arch/arm64/src/common/arm64_backtrace.c

```diff
@@ -143,8 +143,8 @@ int up_backtrace(struct tcb_s *tcb,
             {
               ret += backtrace(rtcb->stack_base_ptr,
                                rtcb->stack_base_ptr + rtcb->adj_stack_size,
-                               (void *)up_current_regs()[REG_X29],
-                               (void *)up_current_regs()[REG_ELR],
+                               (void *)((uint64_t *)running_regs())[REG_X29],
```

Comment:
should we let's running_regs return `void *` or `uinptr_t` to avoid the casting.

Reply:
ok